pandas/tests/plotting/common.py

```diff
@@ -1,3 +1,9 @@
+"""
+Module consolidating common testing functions for checking plotting.
+
+Currently all plotting tests are marked as slow.
```

Comment:
can you add via the `pytest.mark.slow` at the module level

Reply:
Sure! Done.